### PR TITLE
Update Insolation.jl

### PR DESCRIPTION
**Pull request by charleskawczynski**
This PR updates to the latest version of Insolation.jl

### CODE REVIEW

**Review by valeriabarra — APPROVED** [review]

File: src/parameterizations/radiation/radiation.jl [hunk 1]
```diff
@@ -63,7 +63,8 @@ function radiation_model_cache(
         latitude = RRTMGPI.field2array(zero(bottom_coords.z)) # flat space is on Equator
     end
     local radiation_model
-    orbital_data = Insolation.OrbitalData()
+    data_dir = joinpath(pkgdir(CAP), "artifacts")
```
Comment (navidcy) [inline src/parameterizations/radiation/radiation.jl:66]: this was required as part of the update or is just an additional enhancement?

**Review by navidcy — APPROVED** [review]
lgtm

File: src/parameterizations/radiation/radiation.jl (hunk 1 above)
Comment (charleskawczynski) [reply src/parameterizations/radiation/radiation.jl:66]: This was required, related issue: https://github.com/CliMA/ArtifactWrappers.jl/issues/12

**Comment by charleskawczynski** [comment]
Superseded by #1674

**Pull request closed without merging**